Auth0/OAuth2Grant.swift

```diff
@@ -30,11 +31,21 @@ protocol OAuth2Grant {
 
 struct ImplicitGrant: OAuth2Grant {
 
+    let authentication: Authentication
     let defaults: [String: String]
     let responseType: [ResponseType]
-
-    init(responseType: [ResponseType] = [.token], nonce: String? = nil) {
+    let leeway: Int
+    let maxAge: Int?
+
+    init(authentication: Authentication,
+         responseType: [ResponseType] = [.token],
```

Comment:
**Question:** Can you tell me what this is doing? Very unique syntax for me!

Reply:
This is declaring a parameter of type `[ResponseType]`, which means 'array of `ResponseType`', and setting a default value of `[.token]`.